app/models/casa_case.rb

```diff
@@ -63,6 +63,7 @@ def update_cleaning_contact_types(args)
 #  id                     :bigint           not null, primary key
 #  birth_month_year_youth :datetime
 #  case_number            :string           not null
+#  court_report_submitted :boolean          default(FALSE), not null
```

Comment:
Note for the future: since there will be a court report every six months per case, eventually we will probably want to migrate this to its own table. I like this as an excellent lean start!